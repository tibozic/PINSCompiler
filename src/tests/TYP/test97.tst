fun f(x: logical, y: integer): logical = x&y
