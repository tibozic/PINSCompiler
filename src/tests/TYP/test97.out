[1:42-1:45]: ERROR: Types for left and right side of expression don't match: `log` != `int`

